### DISCUSSION TITLE
Compilation of a custom FW for ESP32-cam fails

### DISCUSSION THREAD

**Discussion by KaiBuesing** [arendst/Tasmota#21022] [Q&A]
Hi, I am currently trying to compile my own firmware for an ESP32-cam. It looks quite good so far, but at the end there is an error message. Can someone tell me what I am doing wrong?
<img width="1552" alt="2024-03-24_14h40_17" src="https://github.com/arendst/Tasmota/assets/113059963/5d206c70-383f-4c5e-8bcf-ad4c6d72df3f">

**Comment by Jason2866** [comment]
You can't define scripter functions when using rules. Decide rules OR scripter

**Reply by KaiBuesing** [reply]
My mistake, thanks a lot Jason ! 🙂
